### DISCUSSION TITLE
“社区布景主题”条目内容出错

### DISCUSSION THREAD

**Discussion by oflg** [BramChen/tw5-docs#4] [Q&A] [answered]
内容中多了“天前”字段。

![image](https://user-images.githubusercontent.com/42492105/157242497-3853f7c2-4a06-4425-8a1c-7ee35796cf9c.png)

本想修改后 PR ，但没有找到对应仓库。所以在这里说明，并顺便问下可以在哪里提交中文文档翻译的更正。

**Comment by BramChen** [comment, marked as answer]
非常感謝您。時間有點久了，雖然不記得有這個問題且曾經修正過，但它應該已經不存在了。通常，這裡的文件會緊跟著原文的來源倉庫更新 (不是即時，但一般都是一兩天內會跟上)。
